anni/src/subcommands/repo.rs

```diff
@@ -172,23 +173,13 @@ pub struct RepoGetAction {
 pub enum RepoGetSubcommand {
     #[clap(name = "vgmdb")]
     VGMdb(RepoGetVGMdb),
+    #[clap(name = "cue")]
+    Cue(RepoGetCue),
 }
 
-#[derive(Parser, Debug, Clone)]
-pub struct RepoGetVGMdb {
-    #[clap(short = 'c', long)]
-    catalog: String,
-
-    #[clap(short = 'k', long)]
-    keyword: Option<String>,
-}
-
-#[handler(RepoGetVGMdb)]
-fn repo_get_vgmdb(options: &RepoGetVGMdb, manager: &RepositoryManager, get: &RepoGetAction) -> anyhow::Result<()> {
-    let catalog = &options.catalog;
-
+async fn search_album(keyword: &str) -> anyhow::Result<Album> {
     let client = VGMClient::default();
-    let search = client.search_albums(&options.keyword.as_deref().unwrap_or(catalog)).await?;
+    let search = client.search_albums(keyword).await?;
```

Comment:
Catalog returned by `client.search_albums` might be None. Code below might need to be changed.